building.md

```diff
@@ -4,7 +4,7 @@ The notation repo contains the following:
 
 - `notation` - A CLI for signing and verifying artifacts with Notation
 
-Building above binaries require [golang](https://golang.org/dl/) with version `>= 1.20`.
+Building above binaries require [golang](https://golang.org/dl/) with version `>= 1.21`.
```

Comment:
do we need >= 1.22?

Reply:
1.21 should still be compatible so I included it in the range.

Reply:
The `go.mod` file has been updated to 1.22. Why can it still support 1.21?

Reply:
Make sense. Updated.